tt_metal/jit_build/genfiles.cpp

```diff
@@ -331,6 +331,52 @@ static void generate_data_format_descriptors(JitBuildOptions& options, const tt:
     emit_pack_data_formats(pack_data_format_descs, pack_src_formats_all_cbs, pack_dst_formats_all_cbs);
 }
 
+static std::string array_to_string(const uint32_t arr[]) {
+    std::string formats_string = "";
```

Comment:
I remember we talked a bit about increase in latency in compilation with support for different tile sizes. Do you think this function contributes to it or is negligible ? If it does , we could optimize this one, although I think the writing to files will consume many more cycles than this.

Reply:
should be ok, please note this is following the similar style as the current code, so in terms of perf there won't be much diff.